tools/docker-builder/docker.go

```diff
@@ -173,141 +195,150 @@ func createBuildxBuilderIfNeeded(a Args) error {
 		c.Stdout = out
 		err := c.Run()
 		if err != nil {
-			return fmt.Errorf("command failed: %v", err)
+			return "", fmt.Errorf("command failed: %v", err)
 		}
 		matches := regexp.MustCompile(`Driver:\s+(.*)`).FindStringSubmatch(out.String())
 		if len(matches) == 0 || matches[1] != "docker-container" {
-			return fmt.Errorf("the docker buildx builder is not using the docker-container driver needed for .save.\n" +
+			return "", fmt.Errorf("the docker buildx builder is not using the docker-container driver needed for .save.\n" +
 				"Create a new builder (ex: docker buildx create --driver-opt network=host,image=gcr.io/istio-testing/buildkit:v0.11.0" +
 				" --name container-builder --driver docker-container --buildkitd-flags=\"--debug\" --use)")
 		}
-		return nil
+		return "container-builder", nil
 	}
-	return exec.Command("sh", "-c", `
+	return "container-builder", exec.Command("sh", "-c", `
```

Comment:
What is this return string corresponding to

Reply:
Well windows is sort of a special case, so we'll probably only need 2 builders for the foreseeable future